client/src/components/Home/index.js

```diff
@@ -1,17 +1,9 @@
-import React, { Component } from 'react'
-import { Link } from 'react-router-dom'
-import SignIn from '../SignIn'
-import M from 'materialize-css'
```

Comment:
need this guy

Reply:
this is added back in via a new commit. this is only needed so that the signup modal opens when navigating from any page.